pkgs/applications/misc/rofimoji/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitHub, rofi, xdotool
+, xsel, python3, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "rofimoji";
+  version = "unstable-2020-01-12";
```

Comment:
They have done releases, is there a specific reason to not use a stable release?